picoscope/ps4000.py

```diff
@@ -142,6 +142,18 @@ def __init__(self, serialNumber=None, connect=True):
             self.lib = windll.LoadLibrary(str(self.LIBNAME + ".dll"))
 
         super(PS4000, self).__init__(serialNumber, connect)
+        # check to see which model we have and use special functions if needed
+
+        unit_number = self._lowLevelGetUnitInfo('VariantInfo')
```

Comment:
sorry, this line won't work. it is `getUnitInfo` since the low level functions don't take the *string like* variables

Reply:
k, one sec